index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+    <header class="nav">
+        <a href="" class="m-logo" >My Logo</a>
+         <nav>
+           <ul class="header-link">
+             <li class="link-right">
+                <a href="/someurl" class="inline"><b>Portfolio</b></a>
+                <a href="/someurl" class="inline"><b>About</b></a>
+                <a href="/someurl" class="inline"><b>Contact Us</b></a>
+             </li>
+           </ul>
+        </nav>
+    </header>
```

Comment:
Kindly make this section resemble Figma design as closely as possible for both mobile and desktop 🤗

Your design:
![image](https://user-images.githubusercontent.com/73714615/187713096-ee7b71a3-0ad1-478b-9234-032769986972.png)
![image](https://user-images.githubusercontent.com/73714615/187713185-44b81de8-bc67-4cda-bfc9-add43ce89d27.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187713231-0984c607-4f38-4e18-8be9-c19d80717768.png)
![image](https://user-images.githubusercontent.com/73714615/187713281-4f973b07-0db4-4905-9389-f4c7ea122eae.png)

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉